### PR TITLE
Add intro and more details

### PR DESCRIPTION
**Pull request by lognaturel**
@shobhitagarwal1612 @yanokwa please add commits to this pull request if you want to make changes and/or additions! Or anyone else!!

### CODE REVIEW

**Comment by lognaturel** [comment]
@yanokwa @shobhitagarwal1612 This is ready for final review!

File: drafts/2017-05-09-behind-the-repos.md [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
-- New release of Briefcase on May 28th, with a freeze on May 24th. We'll be adding [network proxy support](https://github.com/opendatakit/briefcase/issues/86), [exporting in a date range](https://github.com/opendatakit/briefcase/issues/117) and [speeding up the app's launch](https://github.com/opendatakit/briefcase/issues/82)
-- Aiming for a new release of [JavaRosa](https://github.com/opendatakit/javarosa), the core form parsing library that powers many ODK tools
-- Refine UI/UX proposal for Collect and start migration to modern UI libraries
+- Refine UI/UX proposal for Collect and start migration to modern UI libraries.
+
+Releases on May 28th with a feature freeze on May 24th
+- Collect: [Track the progress of the May Release](https://github.com/opendatakit/collect/milestone/3). Additions will include [timing](https://github.com/opendatakit/collect/issues/257) and [bulk configuration with QR codes](https://github.com/opendatakit/collect/issues/781)
```
Comment (yanokwa) [inline drafts/2017-05-09-behind-the-repos.md]: I would remove the tracking of the progress. I don't think people care. And if you do, the additions should be lowercase.

File: drafts/2017-05-09-behind-the-repos.md [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
 
 This month
-- New release of Collect on May 28th. We’ll freeze code contributions on May 24th. [Track the progress of the May Release](https://github.com/opendatakit/collect/milestone/3). Additions will include [timing](https://github.com/opendatakit/collect/issues/257) and [bulk configuration with QR codes](https://github.com/opendatakit/collect/issues/781)
-- New release of Briefcase on May 28th, with a freeze on May 24th. We'll be adding [network proxy support](https://github.com/opendatakit/briefcase/issues/86), [exporting in a date range](https://github.com/opendatakit/briefcase/issues/117) and [speeding up the app's launch](https://github.com/opendatakit/briefcase/issues/82)
-- Aiming for a new release of [JavaRosa](https://github.com/opendatakit/javarosa), the core form parsing library that powers many ODK tools
-- Refine UI/UX proposal for Collect and start migration to modern UI libraries
+- Refine UI/UX proposal for Collect and start migration to modern UI libraries.
+
+Releases on May 28th with a feature freeze on May 24th
```
Comment (yanokwa) [inline drafts/2017-05-09-behind-the-repos.md]: I'd move this to the first subsection, called `Releases`, with first bullet point `Releases on May 28th with a feature freeze on May 24th`

File: drafts/2017-05-09-behind-the-repos.md [hunk 3]
```diff
@@ -22,5 +29,6 @@ Feedback needed
 Quick way to contribute
 - Join the [Collect beta program](https://play.google.com/apps/testing/org.odk.collect.android) and provide feedback on pre-release builds. Betas go out on the last Wednesday of the month for a Sunday release.
 
+Thanks and go team ODK!
 
-P.S. < Fun article or video. Shobhit, any ideas? >
\ No newline at end of file
+P.S. Looking for a new webcomic to follow? Shivam suggests [Geek & Poke](http://geek-and-poke.com/)!!
```
Comment (yanokwa) [inline drafts/2017-05-09-behind-the-repos.md]: Single ! please.

File: drafts/2017-05-09-behind-the-repos.md [hunk 4]
```diff
@@ -1,11 +1,18 @@
...
-- Aiming for a new release of [JavaRosa](https://github.com/opendatakit/javarosa), the core form parsing library that powers many ODK tools
-- Refine UI/UX proposal for Collect and start migration to modern UI libraries
+- Refine UI/UX proposal for Collect and start migration to modern UI libraries.
+
+Releases on May 28th with a feature freeze on May 24th
+- Collect: [Track the progress of the May Release](https://github.com/opendatakit/collect/milestone/3). Additions will include [timing](https://github.com/opendatakit/collect/issues/257) and [bulk configuration with QR codes](https://github.com/opendatakit/collect/issues/781)
+- Briefcase: additions will include [network proxy support](https://github.com/opendatakit/briefcase/issues/86), [exporting in a date range](https://github.com/opendatakit/briefcase/issues/117) and [speeding up the app's launch](https://github.com/opendatakit/briefcase/issues/82)
+- [JavaRosa](https://github.com/opendatakit/javarosa): the core form parsing library that powers many ODK tools
```
Comment (yanokwa) [inline drafts/2017-05-09-behind-the-repos.md]: Maybe a word or two about what will be added to JR.

Comment (lognaturel) [reply drafts/2017-05-09-behind-the-repos.md]: I'll let you take this one because I'm not sure what level of detail makes sense.

**Comment by lognaturel** [comment]
So I think @yanokwa will make last changes and send!

**Pull request merged**